docs/index.md

```diff
@@ -6,27 +6,21 @@ description: |-
   
 ---
 
-# Lightstep Provider
+# lightstep Provider
```

Comment:
Were these changes intentional?

Reply:
Weird. Not intentional, but this was autogenerated by the `tfplugindocs` command.

Reply:
Sorry to be all "last one who touched it owns it" on you, but would you mind checking to see where this changed and try to verify whether it was intentional?

Reply:
Looks like you can add doc template files to override some of this. These template files also allow us to keep the examples in the autogenerated file without needed to manually modify it after the fact. I'm going to add templates for the small set of files that we care about for this PR, and then follow up with another PR to fix the other doc files